### PR TITLE
These are files for "Add 'eye' feature on password" in Google Code In

### PR DESCRIPTION
**Pull request by NumberC**
Fixes #
Error #549

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <link rel="stylesheet" href="indexStyle.css">
+    </head>
+    <body>
+        <div id="wrapper">
+            <form>
+                <label for="inUsername">Username</label>
+                <input type="text" id="inUsername">
+                <label for="formPassword">Password</label>
+                <input type="password" id="formPassword">
+                <img src="EyeClosed.png" id="hideOrShow" onclick="toggle()">
```
[the remark this answers is not in the document]
Comment (NumberC) [reply index.html]: Is it better now? I added `alt="Hide Password"` or `alt="Reveal Password`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <link rel="stylesheet" href="indexStyle.css">
+    </head>
+    <body>
+        <div id="wrapper">
+            <form>
+                <label for="inUsername">Username</label>
+                <input type="text" id="inUsername">
+                <label for="formPassword">Password</label>
+                <input type="password" id="formPassword">
+                <img src="EyeClosed.png" id="hideOrShow" onclick="toggle()" alt="Reveal Password">
```
Comment (ankitkataria) [inline index.html]: You have to make these changes in `app/views/users/sessions/new.html.erb`

File: indexStyle.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+:root{
```
Comment (ankitkataria) [inline indexStyle.css]: You can move this in `public/assets/stylesheets/` and use only appropriate class names and ids.

**Comment by ankitkataria** [comment]
Could you mention you GCI issue here @NumberC ?

**Comment by NumberC** [comment]
"Add 'eye' feature on password - CircuitVerse.org"
https://codein.withgoogle.com/dashboard/task-instances/4964929971421184/

File: index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (ankitkataria) [inline index.html]: We can remove this now.

File: indexStyle.css (hunk 3 above)
Comment (ankitkataria) [reply indexStyle.css]: We can remove this file as well.

File: index.html (hunk 4 above)
Comment (NumberC) [reply index.html]: Thank you for reminding me! I fixed it.

**Comment by NumberC** [comment]
Sure, it's nightime in my timezone, so I'll have to do it tomorrow. For this and future instances, I am new to Ruby on Rails and was wondering how I could test this locally. Thank you!

**Comment by ankitkataria** [comment]
No issues @NumberC ! :smile: 
You can use Gitpod for development, explore the [gitpod wiki](https://github.com/CircuitVerse/CircuitVerse/wiki/Development-on-Gitpod) to do so. The Gitpod instance of your code should be available [here](https://gitpod.io/#https://github.com/CircuitVerse/CircuitVerse/pull/679).

**Comment by NumberC** [comment]
@ankitkataria Thank you for the link! The styling issues are fixed and the files are in their correct directories.

**Comment by ankitkataria** [comment]
Awesome work @NumberC ! :tada: 
Approving this task for GCI. 
Adding a screenshot here for others to review before this is merged.

image

**Comment by NumberC** [comment]
Sorry! I accidentally commited changes meant for another branch in this one. I reverted back, though.

**Comment by NumberC** [comment]
I will be refactoring my fork, so I will be closing this PR and opening a new one. The result will be the same, though

**Comment by NumberC** [comment]
Nevermind

**Comment by aayush-05** [comment]
Can be closed @tachyons

**Pull request closed without merging**